ui/widgets/datepicker.js

```diff
@@ -61,12 +62,13 @@ var widget = $.widget( "ui.datepicker", {
 
 	_create: function() {
 		this.suppressExpandOnFocus = false;
+		this._parse = new Globalize( this.options.locale ).dateParser( this.options.dateFormat );
 
 		if ( $.type( this.options.max ) === "string" ) {
-			this.options.max = Globalize.parseDate( this.options.max, { raw: "yyyy-MM-dd" } );
+			this.options.max = this._parse( this.options.max );
```

Comment:
The current implementation is:

1) `min / max` option passed as date
2) `min / max` option passed as string: format value according to `dateFormat` option
3)  `min / max` attributes on input element: format value as `yyyy-MM-dd` (according to HTML specs)

I'm not able to find the reference in the GH comments, but we wanted it this way and I guess we need to update the wiki specs.